File: Java/2206_Divide_Array_Into_Equal_Pairs.java
class Solution {
    public boolean divideArray(int[] nums) {
        boolean result = true;
        int[] t = new int[501];
        for(int num : nums){
            t[num]++;
        }
        for(int i = 1; i <= 500; i++){
            if(t[i] % 2 != 0){
                result = false;
            }
        }
        return result;
    }
}

File: CS/2206_Divide_Array_Into_Equal_Pairs.cs
public class Solution {
    public bool DivideArray(int[] nums) {
        bool[] arr = new bool[500];
        for(int i = 0; i < nums.Length; i++){
            arr[nums[i]-1] = !arr[nums[i]-1];
        }
        for(int i = 0; i < 500; i++){
            if(arr[i]){
                return false;
            }
        }
        return true;
    }
}
